build/Common.nonprod.props

```diff
@@ -20,11 +20,11 @@
       Please sort alphabetically.
       Refer to https://docs.microsoft.com/en-us/nuget/concepts/package-versioning for semver syntax.
     -->
-    <BenchmarkDotNetPkgVer>[0.13.2,0.14)</BenchmarkDotNetPkgVer>
-    <CoverletCollectorPkgVer>[3.1.2,4.0.0)</CoverletCollectorPkgVer>
+    <BenchmarkDotNetPkgVer>[0.13.3,0.14)</BenchmarkDotNetPkgVer>
+    <CoverletCollectorPkgVer>[3.2.0,4.0.0)</CoverletCollectorPkgVer>
     <DotNetXUnitCliVer>[2.3.1,3.0)</DotNetXUnitCliVer>
     <MicrosoftExtensionsLoggingPkgVer>[5.0.0,7.0)</MicrosoftExtensionsLoggingPkgVer>
-    <MicrosoftNETTestSdkPkgVer>[16.11.0,17.0)</MicrosoftNETTestSdkPkgVer>
+    <MicrosoftNETTestSdkPkgVer>[17.3.2,18.0)</MicrosoftNETTestSdkPkgVer>
```

Comment:
Is there any reason to not reference [17.4.1](https://www.nuget.org/packages/Microsoft.NET.Test.Sdk/17.4.1)?

Reply:
We still have some test projects that run on net452. The package version 17.4.1 is not supported on net452.

The package version 17.3.2 is the latest stable version that is supported on net452.